lib/openhab/dsl/rules/automation_rule.rb

```diff
@@ -76,8 +78,8 @@ def execute!(mod, inputs)
             logger.trace { "Execute called with mod (#{mod&.to_string}) and inputs (#{inputs.inspect})" }
             logger.trace { "Event details #{inputs["event"].inspect}" } if inputs&.key?("event")
             trigger_conditions(inputs).process(mod: mod, inputs: inputs) do
-              event = extract_event(inputs)
-              process_queue(create_queue(event), mod, event)
+              @event = extract_event(inputs)
```

Comment:
why does event need to be an ivar now?

Reply:
good catch! This was required in an earlier design. It can be converted back to non ivar now.